### DISCUSSION TITLE
Copilot suggestions disappear after typing space

### DISCUSSION THREAD

**Discussion by cad0p** [community/community#61656] [Copilot Conversations]
### Select Topic Area

Bug

### Body

https://github.com/community/community/assets/14107574/974da4f9-ce0b-4977-a4e3-ad93e2b5ef04

I have set up a shortcut to fill only one word, but when it gets to the space then the suggestion disappears

**Comment by cad0p** [comment]
Another issue is that sometimes the suggestions disappear after only appearing for a brief period of time, but couldn't reproduce

**Comment by LevYas** [comment]
disappearing after appearing happens to me too. Any pointers?
